nims-notes-things/src/components/notes.js

```diff
@@ -1,37 +1,95 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
 import { useSelector } from "react-redux";
+import Pin from "../pin.jpeg";
+import Button from "./Button";
 
 function Notes() {
   const notes = useSelector((state) => {
     return state.noteReducer.value;
   });
 
+  const [sorted, setSorted] = useState(false);
```

Comment:
Told this Alex too, totally up to you, but try this with Redux for setting and handling the state. Would be a good challenge